### PR TITLE
Overpass tile source

### PR DESCRIPTION
**Pull request by Gustl22**
See #663.

### CODE REVIEW

File: vtm-extras/src/org/oscim/tiling/source/overpass/OverpassTileSource.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+import org.oscim.tiling.ITileDataSource;
+import org.oscim.tiling.OverzoomTileDataSource;
+import org.oscim.tiling.source.UrlTileDataSource;
+import org.oscim.tiling.source.UrlTileSource;
+import org.oscim.utils.overpass.OverpassAPIReader;
+
+public class OverpassTileSource extends UrlTileSource {
+
+    private static final String DEFAULT_URL = "https://www.overpass-api.de/api/interpreter?data=[out:json];";
+    private static final String DEFAULT_PATH = "(node{{bbox}};way{{bbox}};>;);out%20body;";
```
Comment (boldtrn) [inline vtm-extras/src/org/oscim/tiling/source/overpass/OverpassTileSource.java]: So this won't fetch relations? Does this mean there are no boundaries and potentially no shorelines?

Comment (Gustl22) [reply vtm-extras/src/org/oscim/tiling/source/overpass/OverpassTileSource.java]: I have not yet tested relations. They have to be managed (e.g. like in mapsforge writer) and I wanted to provide a quick and simple overpass source. So some elements might not be classified correctly. (E.g. in my case boundaries and coastlines were drawn, but they also can vanish).
But anyone is welcomed to extend preprocessing.
As mentioned in #663 would be good to add custom filters/queries in 'Overpass QL'.

Comment (boldtrn) [reply vtm-extras/src/org/oscim/tiling/source/overpass/OverpassTileSource.java]: Ok thanks, I was just wondering :)

**Comment by devemux86** [comment]
Thanks, merged via 23d65486e67d3fb9f2c38be80d125469d6ddb595.

**Pull request closed without merging**